src/components/Proposal/ProposalData.tsx

```diff
@@ -79,10 +80,14 @@ class ProposalData extends React.Component<IProps, IState> {
     // Don't schedule timeout if its too long to wait, because browser will fail and trigger the timeout immediately
     const millisecondsUntilExpires = closingTime(this.props.data[0]).diff(moment());
     if (!this.state.expired && millisecondsUntilExpires < 2147483647) {
-      setTimeout(() => { this.setState({ expired: true });}, millisecondsUntilExpires);
+      this.expireTimeout = setTimeout(() => { this.setState({ expired: true });}, millisecondsUntilExpires);
```

Comment:
I don't see a reason for this.  `setState` is mean to be invoked within `componentDidMount`.

Reply:
Are you not able to consistently reproduce accoring to the lin [given here](https://github.com/daostack/alchemy/issues/1374#issuecomment-583672201)?

Reply:
@dkent600 what do you mean? this is inside componentDidMount

Reply:
and no I was not able to reproduce. I saw the error happen one time but I dont know when.